packages/optimistic-oracle-v2/src/utils/helpers/optimisticOracle.ts

```diff
@@ -17,6 +17,7 @@ export function getOrCreateOptimisticPriceRequest(
     request.currency = Bytes.fromI32(0);
     request.reward = BIGINT_ZERO;
     request.finalFee = BIGINT_ZERO;
+    request.lastUpdated = BIGINT_ZERO;
```

Comment:
Should we init the rest of the packages to Big_int zero for consistency? Or can we drop it since lastUpdate gets set on every PriceRequested event?